packages/generator-adp/src/translations/generator-adp.i18n.json

```
@@ -33,6 +37,8 @@
         "projectFolderPath": "Project Path",
         "ui5VersionLabel": "SAPUI5 Version",
         "ui5VersionTooltip": "Select the SAPUI5 version you want to use to preview your app variant.",
+        "addDeployConfig": "Add deployment configuration",
+        "addFlpConfig": "Add FLP configuration",
```

```suggestion
        "addFlpConfig": "Add FLP configuration",
```
Is this a title,  a button, placeholder?

---

It is a label for the confirm prompt: Yes / No

---

Changed in: [b8a4488](https://github.com/SAP/open-ux-tools/pull/3249/commits/b8a44880c058526a7c1f1a0f54dc5d5ca4a02569)